crates/mdbook-html/front-end/templates/index.hbs

```diff
@@ -10,6 +10,9 @@
         {{#if base_url}}
         <base href="{{ base_url }}">
         {{/if}}
+        {{#if canonical_url}}
+        <link rel="canonical" href="{{ canonical_url }}">
```

Comment:
What do you think about adding also [the Open Graph URL](https://ogp.me/#metadata) ?

```hbs
<meta property="og:url" content="{{ canonical_url }}" />
```

Reply:
I guess the more interesting question is would you want to rewrite social media without also hinting the search engines?
If that's a use case, maybe we should only define `canonical_url` but leave the socials-vs-search decision up to the custom HTML template, to keep things simple for the mdBook implementation?
If we don't have such a use case, I'd like to keep this built-in behavior right here, because otherwise you have a weird setting without any effect unless you customize your theme. That seems like an step too much for me in terms of UX.
So right back at ya, I guess :)
What do you think of removing `<link rel="canonical" .../>`?